src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -934,18 +958,24 @@ class CheatsWindow final : public Window
         }
         if (widgetIndex == WIDX_STAFF_SPEED_DROPDOWN_BUTTON)
         {
-            int32_t speed = CHEATS_STAFF_FAST_SPEED;
+            int32_t speed;
             switch (dropdownIndex)
             {
                 case 0:
+                    gCheatsSelectedStaffSpeed = StaffSpeedCheat::FROZEN;
                     speed = CHEATS_STAFF_FREEZE_SPEED;
                     break;
+
                 case 1:
+                    gCheatsSelectedStaffSpeed = StaffSpeedCheat::NONE;
                     speed = CHEATS_STAFF_NORMAL_SPEED;
-            }
+                    break;
 
+                default:
```

Comment:
Don't use default for enum class types, handle each case explicitly. The compiler will emit a warning for when a case is not handled then.